src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -113,6 +113,99 @@ cssPrefix: pf-v6-c-code-editor
 {{/code-editor}}
 ```
 
+
+### With optional code editor container 
+This is an extra container used in React to prevent event propagation if upload is enabled or there is a provided empty state.
+
+```hbs 
+{{#> code-editor}}
+  {{#> code-editor-container}}
+  {{#> code-editor-header}}
+    {{#> code-editor-header-content}}
+      {{#> code-editor-controls}}
+        {{> code-editor-controls--buttons}}
+      {{/code-editor-controls}}
+    {{/code-editor-header-content}}
+    {{#> code-editor-tab}}
+      {{#> code-editor-tab-icon}}
+        <i class="fas fa-code"></i>
+      {{/code-editor-tab-icon}}
+      {{#> code-editor-tab-text}}
+        HTML
+      {{/code-editor-tab-text}}
+    {{/code-editor-tab}}
+  {{/code-editor-header}}
+  {{#> code-editor-main}}
+    {{#> code-editor-upload}}
+      {{#> empty-state}}
+        {{#> empty-state-header}}
+          {{> empty-state-icon empty-state-icon--type="code"}}
+          {{#> empty-state-title}}
+            {{#> empty-state-title-text}}
+              Start editing
+            {{/empty-state-title-text}}
+          {{/empty-state-title}}
+        {{/empty-state-header}}
+
+        {{#> empty-state-body}}
+          Drag a file here or browse to upload.
+        {{/empty-state-body}}
+
+        {{#> empty-state-footer}}
+          {{#> empty-state-actions}}
+            {{#> button button--IsPrimary=true}}
+              Browse
+            {{/button}}
+          {{/empty-state-actions}}
+          {{#> empty-state-actions}}
+            {{#> button button--IsLink=true}}
+              Start from scratch
+            {{/button}}
+          {{/empty-state-actions}}
+        {{/empty-state-footer}}
+      {{/empty-state}}
+    {{/code-editor-upload}}
+  {{/code-editor-main}}
+  {{/code-editor-container}}
+{{/code-editor}}
+```
+
+
+### With full height modifiers
+
+```hbs
+{{#> code-editor code-editor--modifier="pf-m-full-height"}}
```

Comment:
For this example, can you add a local style to modify the height of the example so that the code editor grows to fill the available space? 

You'll want to create a new file here (in the examples dir) called `CodeEditor.css` and then add `import './CodeEditor.css'` to the top of this MD file - below the metadata and above the first heading, ~L6. Here's an example of this in our [table demo markdown](https://github.com/patternfly/patternfly/blob/5006992758d980967cdcde91213692eda8cedeb9/src/patternfly/demos/Table/examples/Table.md?plain=1#L7) file.

Then in `CodeEditor.css` add something like this:

```
#ws-core-c-code-editor-with-full-height-modifiers .ws-preview-html {
  height: 400px;
}